### PR TITLE
ARM state bag constants.

### PR DESCRIPTION
**Pull request by boumenot**

### CODE REVIEW

File: packer/builder/azure/common/constants/stateBag.go [hunk 1]
```diff
@@ -7,25 +7,37 @@ package constants
...
+	ArmComputeName                     string = "arm.ComputeName"
+	ArmDeploymentName                  string = "arm.DeploymentName"
+	ArmIsValid                         string = "arm.IsValid"
+	ArmLocation                        string = "arm.Location"
+	ArmOSDiskVhd                       string = "arm.OSDiskVhd"
+	ArmPublicIPAddressName             string = "arm.PublicIPAddressName"
+	ArmResourceGroupName               string = "arm.ResourceGroupName"
+	ArmTemplateParameters              string = "arm.TemplateParameters"
+	ArmVirtualMachineCaptureParameters string = "arm.VirtualMachineCaptureParameters"
 )
```
Comment (paulmey) [inline packer/builder/azure/common/constants/stateBag.go]: this file looks like it now needs a `go fmt`?

**Comment by paulmey** [comment]
What do you think, should we split the non-shared constants to the separate builder packages?

**Comment by boumenot** [comment]
I think it makes sense, but lets wait for the dust settle to figure out what's shared and what isn't.  If we _namespace_ them I would be leaving it as is too.

File: packer/builder/azure/common/constants/stateBag.go (hunk 1 above)
Comment (boumenot) [inline packer/builder/azure/common/constants/stateBag.go]: Nope, this one I actually managed to do.

**Pull request merged**